variantlib/commands/config/show.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+
+import tomlkit
+import tomllib
+
+from variantlib.configuration import ConfigEnvironments
+from variantlib.configuration import ConfigurationModel
+from variantlib.configuration import VariantConfiguration
+from variantlib.configuration import get_configuration_files
+from variantlib.errors import ConfigurationError
+
+logging.getLogger("variantlib").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+
+def show(args: list[str]) -> None:
+    parser = argparse.ArgumentParser(
+        prog="show",
+        description="CLI interface to interactively show active configuration.",
+    )
+
+    parser.add_argument(
+        "environment",
+        nargs="?",
+        choices=[x.name for x in ConfigEnvironments],
+        help=(
+            "Optionally specify the exact configuration you wish to see. By default "
+            "`variantlib` will show the active configuration."
+        ),
+        default=None,
+    )
+
+    parsed_args = parser.parse_args(args)
+
+    # note: due to default= above, parsed_args.environment will never be None
+    # however, argparse will still prevent the user from using `-p` and `-e`
+    # simultaneously
+    source_f = None
+    if parsed_args.environment is None:
+        configuration = VariantConfiguration.get_config()
+        config_files = get_configuration_files()
+
+        for config_name in ConfigEnvironments:
+            if (cfg_f := config_files[config_name]).exists():
+                source_f = cfg_f
+                break
+        else:
+            raise FileNotFoundError("No valid configuration file found.")
```

Comment:
```suggestion
            raise FileNotFoundError("No configuration file found.")
```

Invalid files weren't found either.